resources/plugins/simln/README.md

```diff
@@ -0,0 +1,113 @@
+# SimLN Plugin
+
+## SimLN
+SimLN helps you generate lightning payment activity.
+
+* Website: https://simln.dev/
+* Github: https://github.com/bitcoin-dev-project/sim-ln
+
+## Usage
+SimLN uses "activity" definitions to create payment activity between lightning nodes. These definitions are in JSON format.
+
+SimLN also requires access details for each node; however, the SimLN plugin will automatically generate these access details for each LND node. The access details look like this:
+
+```` JSON
+{
+  "id": <node_id>,
+  "address": https://<ip:port or domain:port>,
+  "macaroon": <path_to_selected_macaroon>,
+  "cert": <path_to_tls_cert>
+}
+````
+
+Since SimLN already has access to those LND connection details, it means you can focus on the "activity" definitions.
+
+### Launch activity definitions from the command line
+The SimLN plugin takes "activity" definitions like so:
+
+`./simln/plugin.py launch-activiy '[{\"source\": \"tank-0003-ln\", \"destination\": \"tank-0005-ln\", \"interval_secs\": 1, \"amount_msat\": 2000}]'"''`
+
+### Launch activity definitions from within `network.yaml`
+When you initialize a new Warnet network, Warnet will create a new `network.yaml` file.  If your `network.yaml` file includes lightning nodes, then you can use SimLN to produce activity between those nodes like this:
+
+<details>
+<summary>network.yaml</summary>
+
+````yaml
+nodes:
+  - name: tank-0000
+    addnode:
+      - tank-0001
+    ln:
+      lnd: true
+
+  - name: tank-0001
+    addnode:
+      - tank-0002
+    ln:
+      lnd: true
+
+  - name: tank-0002
+    addnode:
+      - tank-0000
+    ln:
+      lnd: true
+
+  - name: tank-0003
+    addnode:
+      - tank-0000
+    ln:
+      lnd: true
+    lnd:
+      config: |
+        bitcoin.timelockdelta=33
+      channels:
+        - id:
+            block: 300
+            index: 1
+          target: tank-0004-ln
+          capacity: 100000
+          push_amt: 50000
+
+  - name: tank-0004
+    addnode:
+      - tank-0000
+    ln:
+      lnd: true
+    lnd:
+      channels:
+        - id:
+            block: 300
+            index: 2
+          target: tank-0005-ln
+          capacity: 50000
+          push_amt: 25000
+
+  - name: tank-0005
+    addnode:
+      - tank-0000
+    ln:
+      lnd: true
+
+plugins:
+  postDeploy:
+    simln:
+      entrypoint: "../../plugins/simln"  # This is the path to the simln plugin folder (relative to the network.yaml file).
+      activity: '[{"source": "tank-0003-ln", "destination": "tank-0005-ln", "interval_secs": 1, "amount_msat": 2000}]'
+````
+
+</details>
+
+
+## Generating your own SimLn image
+The SimLN plugin fetches a SimLN docker image from dockerhub. You can generate your own docker image if you choose:
+
+1. Clone SimLN: `git clone git@github.com:bitcoin-dev-project/sim-ln.git`
+2. Follow the instructions to build a docker image as detailed int the SimLn repository.
```

Comment:
need to make sure its cross-arch as well! this one isnt: https://hub.docker.com/r/mplsgrant/sim-ln/tags

Reply:
I'm confused. Did the mplsgrant/sim-ln image work on mac (cross arch) when you switched back to it?

If so, I bet we could just update the official simln image to the latest commit using their instructions.

Reply:
the mplsgrant image did work, docker desktop on macos is emulating x86 in this case i think. Still, its kind of warnet best practice to build for all platforms